### PR TITLE
Critical: Change @id for name to schema.org/name.

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

**Review by davidlehn — APPROVED** [review]
Note the default is the string type.  The explicit name and description string type is not needed (and is not in the official context).

**Review by msporny — APPROVED** [review]

File: contexts/credentials-v2.jsonld [hunk 1]
```diff
@@ -46,7 +46,7 @@
           "@type": "@id"
         },
         "name": {
-          "@id": "https://schema.org/description",
+          "@id": "https://schema.org/name",
           "@type": "http://www.w3.org/2001/XMLSchema#string"
```
Comment (msporny) [inline contexts/credentials-v2.jsonld]:
```suggestion
```

File: js/context.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ module.exports = {
           '@type': '@id'
         },
         name: {
-          '@id': 'https://schema.org/description',
+          '@id': 'https://schema.org/name',
           '@type': 'http://www.w3.org/2001/XMLSchema#string'
```
Comment (msporny) [inline js/context.js]:
```suggestion
```

File: js/context.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ module.exports = {
           '@type': '@id'
         },
         name: {
-          '@id': 'https://schema.org/description',
+          '@id': 'https://schema.org/name',
```
Comment (msporny) [inline js/context.js]:
```suggestion
          '@id': 'https://schema.org/name'
```

File: contexts/credentials-v2.jsonld [hunk 4]
```diff
@@ -46,7 +46,7 @@
           "@type": "@id"
         },
         "name": {
-          "@id": "https://schema.org/description",
+          "@id": "https://schema.org/name",
```
Comment (msporny) [inline contexts/credentials-v2.jsonld]:
```suggestion
          "@id": "https://schema.org/name"
```

**Review by msporny — CHANGES_REQUESTED** [review]
Drop the xsd:string types, not needed, IIRC.

**Pull request merged**